From b1865cedc9a3f8ed68c5dbac3f1193d3235caeb7 Mon Sep 17 00:00:00 2001
From: Pengpeng Cui <pengpengx.cui@intel.com>
Date: Mon, 14 Dec 2015 18:14:37 +0800
Subject: [PATCH 11/13] atomisp2: css2400 and 2401
 irci_ecr-master_20151212_1500

Merge CSS2.1 for css2400 and 2401_legacy
for irci_ecr-master_20151212_1500 firmware tag

Change-Id: I3ded7210c17d8a83f69980642a6f9dfc6ea0e18f
Original-Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-47139
Tracked-On:https://jira01.devtools.intel.com/browse/OAM-13009
Signed-off-by: Pengpeng Cui <pengpengx.cui@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/89833
Approver: WeiX Meng <weix.meng@intel.com>
Build: Dessert M BXT RVP (SC)
Build: Dessert M COHO (SC)
Build: Dessert M CHT FFD (SC)
Integrator: WeiX Meng <weix.meng@intel.com>
Reviewed-by: WeiX Meng <weix.meng@intel.com>
Reviewed-on: https://android.intel.com:443/450392
---
 .../external_drivers/camera/drivers/media/pci/atomisp2/css/ia_css.h   | 2 +-
 .../camera/drivers/media/pci/atomisp2/css/ia_css_types.h              | 2 +-
 .../camera/drivers/media/pci/atomisp2/css/ia_css_version_data.h       | 2 +-
 .../camera/drivers/media/pci/atomisp2/css/sh_css_firmware.c           | 4 ++--
 4 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/ia_css.h b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/ia_css.h
index 8e7e84c..375c72f 100644
--- a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/ia_css.h
+++ b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/ia_css.h
@@ -1,4 +1,4 @@
-/* Release Version: irci_ecr-master_20151208_2222 */
+/* Release Version: irci_ecr-master_20151212_1500 */
 /**
 Support for Intel Camera Imaging ISP subsystem.
 Copyright (c) 2010 - 2015, Intel Corporation.
diff --git a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/ia_css_types.h b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/ia_css_types.h
index bbd95ca..c2ed1c6 100644
--- a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/ia_css_types.h
+++ b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/ia_css_types.h
@@ -1,4 +1,4 @@
-/* Release Version: irci_ecr-master_20151208_2222 */
+/* Release Version: irci_ecr-master_20151212_1500 */
 /**
 Support for Intel Camera Imaging ISP subsystem.
 Copyright (c) 2010 - 2015, Intel Corporation.
diff --git a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/ia_css_version_data.h b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/ia_css_version_data.h
index fc93f71..da20e58 100644
--- a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/ia_css_version_data.h
+++ b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/ia_css_version_data.h
@@ -22,7 +22,7 @@ more details.
 #define __IA_CSS_VERSION_DATA_H
 
 
-#define CSS_VERSION_STRING "REL:20151209_50.3_0448; API:2.1.20.20; GIT:irci___#9f9f22f7c8f6adc59a9dd81b12314b03984d1e68; SDK:; USER:viedifw; "
+#define CSS_VERSION_STRING "REL:20151212_50.6_2020; API:2.1.20.20; GIT:irci___#8f56dd693e4bd755b3c56e1383dc2c9ced11462d; SDK:; USER:viedifw; "
 
 
 #endif
diff --git a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/sh_css_firmware.c b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/sh_css_firmware.c
index b759584..de8140e 100644
--- a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/sh_css_firmware.c
+++ b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css/sh_css_firmware.c
@@ -47,10 +47,10 @@ struct fw_param {
 /* Warning: same order as SH_CSS_BINARY_ID_* */
 static struct firmware_header *firmware_header;
 
-/* The string STR(irci_ecr-master_20151208_2222) is a place holder
+/* The string STR(irci_ecr-master_20151212_1500) is a place holder
  * which will be replaced with the actual RELEASE_VERSION
  * during package generation. Please do not modify  */
-static const char *release_version = STR(irci_ecr-master_20151208_2222);
+static const char *release_version = STR(irci_ecr-master_20151212_1500);
 
 #define MAX_FW_REL_VER_NAME	300
 static char FW_rel_ver_name[MAX_FW_REL_VER_NAME] = "---";
-- 
1.9.1

